docs/pages/repo/docs/_meta.json

```diff
@@ -4,6 +4,7 @@
   },
   "installing": "Installing Turborepo",
   "getting-started": "Getting Started",
+  "debugging": "Debugging Turborepo",
```

Comment:
Should we combine with troubleshooting?

Reply:
I think there is SoC but I'm not sure what the naming should be. "Debugging Builds" may be better or, since most of this is around caching simply 'Debugging Caching'/

Reply:
Good call, I think troubleshooting topics. Of the troubleshooting topics

These two should go somewhere in the Monorepo handbook
- My dependency isn't being built correctly
- My types are not being found

These are relevant to the content here. I'll use the existing trouble shooting page for this content and then move monorepo troubleshooting into the handhook
- I'm not seeing any cache hits
- I'm seeing cache hits, but my build is broken
- My build is caching the wrong environment variables